code/game/objects/items/holy_weapons.dm

```diff
@@ -271,6 +271,10 @@
 		message_admins("[ADMIN_LOOKUPFLW(user)] erased a [target_rune.cultist_name] rune with a null rod.")
 	SSshuttle.shuttle_purchase_requirements_met[SHUTTLE_UNLOCK_NARNAR] = TRUE
 
+// The nullrod by itself does not block cult/wizard teleports
+/obj/item/nullrod/intercept_teleport(channel, turf/origin, turf/destination)
+	return
```

Comment:
This is incredibly hacky, not sending the signal at all because its a nullrod. If you want certain components to block teleports, then it should be a flag on the component itself and should be applied fully. You need to be careful that the wizard hardsuit with magic protection isn't going to block the wizard from teleporting to the station.

Reply:
So as it turns out, I removed the INTERCEPT_TELEPORT for /datum/component/anti_magic, and just used .anti_magic_check() instead with `self = TRUE`, which allows us to exclude stuff on our person (like gem encrusted hardsuit) but include stuff like holymelons.